server/db/migrations/20171118234823-create-users.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
```

Comment:
why are you still pushing migrations files?

Reply:
I think this was before you told us not to,not sure but i've made necessary changes in recent PR's